### PR TITLE
[Fix CI] Add Label Commenter

### LINKED ISSUE #5192: [CI] Add Label Commenter
Current Behavior

Copy/paste [this workflow ](https://github.com/meshery/meshery/blob/master/.github/workflows/label-commenter.yml)and [this configuration](https://github.com/meshery/meshery/blob/master/.github/label-commenter-config.yml) into the .github folder, so that this repo will also have an automated commenter.

### Acceptance Tests

Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

    🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
    🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
    🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io/) and [Layer5 Community Slack](http://slack.layer5.io/)

### PR DESCRIPTION
**Pull request by sandramsc**
**Description**

This PR fixes #5192

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 74b8d42e2a8e504c5a8d375f9a79e4c160b09882 at: https://6573267e13a6c81a5d2b7a4e--layer5.netlify.app

**Comment by abhijeetgauravm** [comment]
Hey @sandramsc  Let's discuss this on website call tomorrow at 6:30 PM IST / 7 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by l5io** [comment]
🚀 Preview for commit dadce5970ba8c8aad01483bc3f1a13472c0824af at: https://657656ab096c7907b8734651--layer5.netlify.app

File: .github/label-commenter-config.yml [hunk 1]
```diff
@@ -2,16 +2,15 @@ comment:
   # header: "Please note the following requirement:"
   footer: "\
     ---\n\n
-    > &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Be sure to [join the community](https://slack.layer5.io), if you haven't yet and please leave a :star: [star on the project](../stargazers) :smile:
+    > &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Be sure to [join the community](https://slack.meshery.io), if you haven't yet and please leave a :star: [star on the project](../stargazers) :smile:
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply .github/label-commenter-config.yml:5]: Thanks for catching this. We have two CNAME records - one on each domain - that both point to the same location. In this way, we're able to customize the URL used depending upon whether the invitation is on a Layer5 or a Meshery repo/web property/presentation.

**Comment by l5io** [comment]
🚀 Preview for commit 0ad859a62d306b3e33618867139860107fbb1cfb at: https://657708ee60517d7ae5c11fd0--layer5.netlify.app

**Pull request merged**